### ISSUE TITLE
Cron job - vtigercron.sh: line 10: $'\r': command, line 18: $'php\r': command not found

### ISSUE THREAD

**Issue by dannypingpong** [YetiForceCompany/YetiForceCRM#6132]
#### Issue

After each time the cron job runs, I got the following error message in my cron.log file:

/home/jktaxca/public_html/wm/cron/vtigercron.sh: line 10: $'\r': command not found
/home/jktaxca/public_html/wm/cron/vtigercron.sh: line 15: $'\r': command not found
/home/jktaxca/public_html/wm/cron/vtigercron.sh: line 16: cd: /home/jktaxca/public_html/wm/cron/..

: No such file or directory
/home/jktaxca/public_html/wm/cron/vtigercron.sh: line 18: $'php\r': command not found

#### Actual Behavior

I have the following cron job scheduled:

sh /home/jktaxca/public_html/wm/cron/vtigercron.sh &>/home/jktaxca/public_html/wm/cache/logs/cron.log

And here is a copy/paste of the vtigercron.sh file (after pasted here I got rid of a space between the comment-out #and the first letter and some bland lines for better viewing here, I've never touched the original vtigercron.sh file on the server). 

#*********************************************************************************
#The contents of this file are subject to the vtiger CRM Public License Version 1.0
#("License"); You may not use this file except in compliance with the License
#The Original Code is:  vtiger CRM Open Source
#The Initial Developer of the Original Code is vtiger.
#Portions created by vtiger are Copyright (C) vtiger.
#All Rights Reserved.
#********************************************************************************

export VTIGERCRM_ROOTDIR=`dirname "$0"`/..
export USE_PHP=php
#export USE_PHP=/usr/local/php53/bin/php
#export USE_PHP=/usr/local/php56/bin/php56
cd $VTIGERCRM_ROOTDIR
#TO RUN ALL CORN JOBS
$USE_PHP -f cron/vtigercron.php 

#### Expected Behavior
Basically there are only 4 lines in this vtigercron.sh file. Unfortunately I just can't get it right. 

#### How to trigger the error
Every time the cron job runs, I got the aforesaid errors. 

#### Your Environment
* YetiForce Version used: 4.3
* Browser name and version: Chrome the latestet

Thanks!

**Comment by dannypingpong** [comment]
When I copied/pasted the vtigercron.sh file here on this Github page, the single quotation marks disappeared. Here is what I have on the server:
http://jkonline.ca/danny/weird.png

**Comment by mariuszkrzaczkowski** [comment]
Check the file encoding, it has to be UNIX as it may not work.

**Comment by dannypingpong** [comment]
@mariuszkrzaczkowski Thanks!

After many hours of head banging and an actual clean re-installation of YetiForce, I still have this problem. 

Yes I have checked the vtigercron.sh file coding, it was already UTF-8 so it shouldn't be a problem. 

Can anyone kindly advise me where I could have missed anything?  This is my vtigercron.sh file:
https://jkonline.ca/danny/already_utf-8.png

**Comment by vovpff** [comment]
@dannypingpong what is server OS?

**Comment by dannypingpong** [comment]
@vovpff It's A2 hosting on Linux. 

https://jkonline.ca/danny/server-info.png
https://jkonline.ca/danny/php-version.png

**Comment by vovpff** [comment]
@dannypingpong no ideas for this time...

**Comment by dannypingpong** [comment]
@vovpff Thanks bud! 

I just found out I can purchase support hours from YetiForce. Okay so let me do this... let me note down the issues I have and I'll make a list, so that when I talk to YetiForce support team, the efficiency can be optimized.

**Comment by mariuszkrzaczkowski** [comment]
You have a few problems

- you have an incorrect end-of-line character and the lines have extra spacing, which means that the files are damaged, you must upload the original with the correct end-line
![already_utf-8](https://user-images.githubusercontent.com/10198654/36171819-53cca4c8-1104-11e8-91de-83bf02ab159c.png)
the file should look like this
https://github.com/YetiForceCompany/YetiForceCRM/blob/4.3.0/cron/vtigercron.sh

- You are using the wrong version of PHP, we recommend 7.1.x
![server-info](https://user-images.githubusercontent.com/10198654/36171721-0b99ed64-1104-11e8-99af-3ec0fa13eb7c.png)

- you configure the extensions for php 7.0 and for currently used
![php-version](https://user-images.githubusercontent.com/10198654/36171950-bcf6df5e-1104-11e8-8933-8085cb611f0c.png)
